### DISCUSSION TITLE
Among Us Error Model

### DISCUSSION THREAD

**Discussion by craftablescience** [StrataSource/Portal-2-Community-Edition#1049] [Resources]
Seen in an old update to P2:CE before it was removed from the game. Extract the contents of this zip file to a folder in p2ce/custom.

[error.zip](https://github.com/ChaosInitiative/Portal-2-Community-Edition/files/9687107/error.zip)

**Comment by JJL772** [comment]
Wow awesome :trollface:

**Comment by Hambrix** [comment]
Yaaaay :D

**Comment by AlexBaker2648** [comment]
i can finally remake my shitpost hammer screenshot!

**Comment by gmodn** [comment]
"Quality of Life"

**Comment by CheeseInSpace** [comment]
yes
